letsencrypt/cli.py

```diff
@@ -351,7 +351,8 @@ def diagnose_configurator_problem(cfg_type, requested, plugins):
             msg = "The requested {0} plugin does not appear to be installed".format(requested)
         else:
             msg = ("The {0} plugin is not working; there may be problems with "
-                   "your existing configuration").format(requested)
+                   "your existing configuration.\nThe error was: {1}"
+                   .format(requested, plugins[requested].problem))
```

Comment:
I think we should also be displaying the exception type, at least while there are issues like #1092 around. Otherwise, people will get the following confusing behavior:
```
$ export TMP=$(mktemp -d)
$ letsencrypt --config-dir $TMP/config --work-dir $TMP/work --logs-dir $TMP/logs --apache -t --agree-dev-preview
2015-10-23 18:01:16,865:WARNING:letsencrypt.cli:Root (sudo) is required to run most of letsencrypt functionality.
The apache plugin is not working; there may be problems with your existing configuration.
The error was: 
$
```
It's a UX question I don't know the answer to whether we should continue doing this once #1092 is fixed, but I think we should at least be doing it for the time being.